templates/Dockerfile-template.vaapi1804

```diff
@@ -39,7 +39,7 @@ RUN      buildDeps="autoconf \
                     libva-dev \
                     zlib1g-dev" && \
         apt-get -yqq update && \
-        apt-get install -yq --no-install-recommends ${buildDeps}
+        DEBIAN_FRONTEND=noninteractive apt-get install -yq --no-install-recommends ${buildDeps}
```

Comment:
not necessary now that we don't have tclsh

Reply:
forgot to remove DEBIAN_FRONTEND=noninteractive. now removed!